src/__stories__/stacking-group-story.tsx

```diff
@@ -31,8 +31,8 @@ export const Default: StoryComponent<Args> = ({
     inverse = false,
 }) => {
     return (
-        <ResponsiveLayout isInverse={inverse} dataAttributes={{testid: 'stacking-group'}}>
-            <Box paddingY={24}>
+        <ResponsiveLayout isInverse={inverse}>
+            <Box padding={16} width="max-content" dataAttributes={{testid: 'stacking-group'}}>
```

Comment:
just set paddingY, the horizontal padding is already managed by ResponsiveLayout